ui/pages/settings/info-tab/info-tab.component.js

```diff
@@ -73,6 +75,25 @@ export default class InfoTab extends PureComponent {
             {t('terms')}
           </Button>
         </div>
+        {isBeta() ? (
+          <div ref={this.settingsRefs[8]} className="info-tab__link-item">
+            <Button
+              type="link"
+              href="https://metamask.io/beta-terms.html"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="info-tab__link-text"
+            >
+              {t('betaTerms')}
+              <Chip
+                borderColor={COLORS.PRIMARY_DEFAULT}
+                backgroundColor={COLORS.PRIMARY_DEFAULT}
```

Comment:
After adding these colors, this Chip looks very similar to our Button components, but it doesn't share the Button behaviors. I think I kind of see why it works here because it also sends the user to the link when clicked on. 

curious if you have any thoughts here regarding this Chip instance @georgewrmarshall

Reply:
Yeah I agree was hesitant about the tag UI for this exact reason. I've asked for some design input from @Akatori-Design [in the Figma file
](https://www.figma.com/file/qn0DJn2Mfw036gdt2aYuZB?node-id=3452:7966#299953888)

<img width="660" alt="Screen Shot 2022-11-04 at 7 37 47 PM" src="https://user-images.githubusercontent.com/8112138/200095409-b51e7c8c-db1c-472b-892c-6eb7a05ed2fb.png">

Reply:
What if we just use the standard tag style?
<img width="287" alt="image" src="https://user-images.githubusercontent.com/55973039/200406636-af8589f4-5a91-4859-9076-77c35b30426d.png">

Reply:
I'm fine with that! We have this component available for use! https://metamask.github.io/metamask-storybook/?path=/docs/ui-components-component-library-tag-tag-stories-js--default-story